package.json

```diff
@@ -142,14 +142,14 @@
 		"progress": "2.0.3",
 		"puppeteer-core": "23.10.1",
 		"raw-loader": "4.0.2",
-		"react": "18.3.1",
+		"react": "19.0.0",
```

Comment:
Do we plan to pin the versions to the latest React (19.1.1)? Ref: https://github.com/facebook/react/releases.